src/OrchardCore/OrchardCore.Email.Core/Services/SmtpService.cs

```diff
@@ -250,6 +252,11 @@ private async Task<string> SendOnlineMessage(MimeMessage message)
                     }
                 }
 
+                if (_options.Proxy != null)
```

Comment:
> Is it null when none of its properties is set?

It should be

> What about flattening the properties in the current class, and adding a bool UseProxy too to make it explicit ?

This make things hard and confusing, especially we already have host & port for SMTP server

Reply:
UseProxy
ProxyHost
ProxyPassword

I am not confused ;)

Reply:
If you prefix them with `Proxy` no problem, but again I don't think there's a need for `UseProxy` if I set the `ProxyHost`, so that's why I don't think there's an issue with the `Proxy` property

Again I'm not against any feedback, but that's my thought from a user perspective

Reply:
Also agree there is no need for `UseProxy` and just use `ProxyHost` value if it's set.